### ISSUE TITLE
Could you provide the loss curve？

### ISSUE THREAD

**Issue by Sierkinhane** [Daniil-Osokin/lightweight-human-pose-estimation.pytorch#10]

**Comment by Daniil-Osokin** [comment]
Here you are:
![train_loss](https://user-images.githubusercontent.com/2491976/55278070-af7c7300-5318-11e9-9b92-4aebc0cc9b46.png)

**Comment by Sierkinhane** [comment]
Thank you！

**Comment by Sierkinhane** [comment]
Have you saved the loss in a file? I want to make a precise comparison with yours.

**Comment by Sierkinhane** [comment]
X-axis is epoch, I'm not sure that whether you have trained ~280 iteration on whole coco data set?

**Comment by Daniil-Osokin** [comment]
There is some randomness in training (in augmentation and batch selection), so I don't believe that it is easy to obtain same loss numbers with mine (or between different experiment runs).

**Comment by Sierkinhane** [comment]
Thank you for your reply！

**Comment by tangfayuan** [comment]
> Here you are:
> ![train_loss](https://user-images.githubusercontent.com/2491976/55278070-af7c7300-5318-11e9-9b92-4aebc0cc9b46.png)
Why the loss value is not used as a percentage and the loss value is high？
